core-api/src/main/java/com/optimizely/ab/config/parser/JsonConfigParser.java

```diff
@@ -69,6 +69,11 @@ public ProjectConfig parseProjectConfig(@Nonnull String json) throws ConfigParse
 
             List<EventType> events = parseEvents(rootObject.getJSONArray("events"));
             List<Audience> audiences = parseAudiences(rootObject.getJSONArray("audiences"));
```

Comment:
Why?

Reply:
So that app backend isn't tied down with `audiences` if we decide to move entirely to `typedAudiences` at some point. Semantically, it seems reasonable to make `audiences` optional once `typedAudiences` is supported too.